src/libraries/Directory.Build.props

```diff
@@ -327,6 +327,7 @@
 
   <PropertyGroup Condition="'$(IsTestProject)' == 'true'">
     <EnableTestSupport>true</EnableTestSupport>
+    <EnableRunSettingsSupport>true</EnableRunSettingsSupport>
```

Comment:
Should we set this to false when in CI, instead of conditioning the targets for that?

Reply:
The target below is already conditioned to not run on CI, I don't think this is necessary.

Reply:
I know, just suggesting to instead disable the whole feature, hard to track when this runs because now I need to go and look at the targets conditions instead of looking at the feature flag.

Reply:
Oh actually I just saw the line that you were commenting on (I thought you were talking about that property in pretest.proj). No we shouldn't disable that flag in CI as we will use it very soon when we switch away from xunit.console.

Reply:
> No we shouldn't disable that flag in CI as we will use it very soon when we switch away from xunit.console.

Hmm, then I think we should have the targets that depend on this flag, conditioned against this flag, and then disable this flag on CI for now, and whenever we enable this on CI just remove the condition on this flag.

Reply:
> Hmm, then I think we should have the targets that depend on this flag, conditioned against this flag, and then disable this flag on CI for now.

That's already the case. If the targets file isn't imported then no targets responsible for the .runsettings generation will run. I will add the condition for the sake of closing this discussion.

Reply:
Thanks, I think it is much cleaner to have one consistent condition to set a flag and then use that flag in other places than having multiple flags affecting the same feature.

Reply:
Not arguing with that.